lib/puppet/provider/jail/pyiocage.rb

```diff
@@ -44,30 +59,38 @@ def self.prefetch(resources)
   end
 
   def self.instances
+    default_properties = get_jail_properties('default')
```

Comment:
I've just updated iocage on my system and still get an error here on reading the 'default' jail, which doesn't exist on my system.